CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ IMPROVEMENTS:
 * metrics: add namespace label as appropriate to metrics [[GH-5847](https://github.com/hashicorp/nomad/issues/5847)]
 
 BUG FIXES:
+* metrics: upgrade prometheus client to avoid label conflicts [[GH-5850](https://github.com/hashicorp/nomad/issues/5850)]
 
 * ui: Fixed ability to click sort arrow to change sort direction [[GH-5833](https://github.com/hashicorp/nomad/pull/5833)]
```

Comment:
style: the new log line should be swapped with the new line.  Looks like we should add a line after "IMPROVEMENTS"  to stay consistent with how other headers are done.

Reply:
thanks!